2022/08/18 19:35:16 [error] 9#9: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.128.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.128.1, server: project-test, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.128.5:9000", host: "project-test.ru"
2022/08/18 19:36:35 [error] 9#9: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.128.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.128.1, server: project-test, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.128.5:9000", host: "project-test.ru"
2022/08/18 19:41:18 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.128.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.128.1, server: project-test, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.128.5:9000", host: "project-test.ru"
2022/08/18 19:44:25 [crit] 7#7: *5 SSL_do_handshake() failed (SSL: error:140944E7:SSL routines:ssl3_read_bytes:reason(1255):SSL alert number 255) while SSL handshaking, client: 192.168.128.1, server: 0.0.0.0:443
2022/08/18 19:44:25 [crit] 7#7: *6 SSL_do_handshake() failed (SSL: error:140944E7:SSL routines:ssl3_read_bytes:reason(1255):SSL alert number 255) while SSL handshaking, client: 192.168.128.1, server: 0.0.0.0:443
2022/08/18 19:44:31 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.128.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.128.1, server: project-test, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.128.5:9000", host: "project-test.ru"
2022/08/18 19:45:03 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.128.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.128.1, server: project-test, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.128.5:9000", host: "project-test.ru"
2022/08/18 19:49:20 [error] 8#8: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.128.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 192.168.128.1, server: project-test, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.128.5:9000", host: "project-test.ru"
2022/08/18 19:49:21 [error] 8#8: *3 FastCGI sent in stderr: "PHP message: [critical] Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\FatalError: "Error: Maximum execution time of 30 seconds exceeded" at /project-test/vendor/symfony/error-handler/DebugClassLoader.php line 1135" while reading upstream, client: 192.168.128.1, server: project-test, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.128.5:9000", host: "project-test.ru"
2022/08/18 19:51:23 [error] 8#8: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 192.168.128.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET https://project-test.ru/"" at /project-test/vendor/symfony/http-kernel/EventListener/RouterListener.php line 128" while reading response header from upstream, client: 192.168.128.1, server: project-test, request: "GET / HTTP/2.0", upstream: "fastcgi://192.168.128.5:9000", host: "project-test.ru"
